### ISSUE TITLE
【Exp Result】Unable to obtain expected results

### ISSUE THREAD

**Issue by xdobetter** [YuliangXiu/PuzzleAvatar#8]
I used the data you provided "yuliang" , but I found that the results I got from running the program were quite different from those shown in the paper. Is the main reason for this the multi-concept DreamBooth? If so, how can I adjust it to reduce the deviation?

https://github.com/user-attachments/assets/e109a5a7-e06d-454a-a4dc-9026acefa427

**Comment by YuliangXiu** [comment]
There are several potential reasons:
- The models called by gpt4o API varies, so you might need to update the [gpt4v_prompt.txt](https://github.com/YuliangXiu/PuzzleAvatar/blob/main/multi_concepts/gpt4v_prompt.txt) accordingly
- It ever occurred that even with the same `gpt4v_simple.json`, the GroundedSAM segmentation results varies, you need to check the packed results after running `island_all.py`, if the face pictures are not enough, the final 3D face cannot be well preserved. If this happens, try to tune the `TEXT_TRESHOLD` and `BOX_TRESHOLD` at [grounding_dino_sam.py#L308](https://github.com/YuliangXiu/PuzzleAvatar/blob/main/multi_concepts/grounding_dino_sam.py#L308) to ensure more segmented faces, and other garment parts are well kept.

**Comment by xdobetter** [comment]
I understand. Thank you very much.
1: So, if I want to reproduce your results, I can skip Step 0 and choose to use your default preprocessed data, right?
2: I find the two parameters https://github.com/YuliangXiu/PuzzleAvatar/blob/e1d46171f3f8cdada56b22d556f2cdfc291f446f/multi_concepts/grounding_dino_sam.py#L308-L309  you gave very difficult to adjust for face. Can you give me some suggestions?

**Comment by YuliangXiu** [comment]
Yes, you can skip step 0 and use my provided proprocessed data. and there are a lot discussions on these parameters in issue of [GroundedSAM]( https://github.com/IDEA-Research/Grounded-Segment-Anything/issues), for example, https://github.com/IDEA-Research/Grounded-Segment-Anything/issues/340

**Comment by xdobetter** [comment]
I have a question about multi-concept DreamBooth training. I found that the training time on different data is quite different, such as yuliang and yamei. Do you know if this is normal?  I trained them individually using the A6000.

![image](https://github.com/user-attachments/assets/8391739c-b4a0-496c-bad7-1f6bd76dd538)
![image](https://github.com/user-attachments/assets/4fe94c37-c1fc-4f81-94e3-565d11450194)

**Comment by YuliangXiu** [comment]
> I have a question about multi-concept DreamBooth training. I found that the training time on different data is quite different, such as yuliang and yamei. Do you know if this is normal? I trained them individually using a single A6000.
> 
> ![image](https://private-user-images.githubusercontent.com/26523963/376994060-8391739c-b4a0-496c-bad7-1f6bd76dd538.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkxMDg2NzEsIm5iZiI6MTcyOTEwODM3MSwicGF0aCI6Ii8yNjUyMzk2My8zNzY5OTQwNjAtODM5MTczOWMtYjRhMC00OTZjLWJhZDctMWY2YmQ3NmRkNTM4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE2VDE5NTI1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5ZTg3YzM2ZTBjZTZlZDMwYWYxZWE5ZDk3N2FlMjZkNWVlNTI0MDc5Y2ViY2Q2N2UwMjI5NGJkOGJiYzc2ZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.Zb3TRb4EzFS6n_-ZyJDtJ1wOsc1NQB-ldl0XH229HMg) ![image](https://private-user-images.githubusercontent.com/26523963/376994225-4fe94c37-c1fc-4f81-94e3-565d11450194.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkxMDg2NzEsIm5iZiI6MTcyOTEwODM3MSwicGF0aCI6Ii8yNjUyMzk2My8zNzY5OTQyMjUtNGZlOTRjMzctYzFmYy00ZjgxLTk0ZTMtNTY1ZDExNDUwMTk0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE2VDE5NTI1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFmMjNkMmZhOWNkMGQwMzIxZmQyODdiMTU3MDJkOTQ2MmI1NThmZTdhMjA5MTU1YzY4N2NkZTcxMjllMDY4YWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.pgmVLHtX0v5lPfZsGum2gWtWUjdHfx93dgUrAGsSO14)

This is not normal, you should check the dataloader if there experiments share the same hyper paramters.
